TOC.md

```diff
@@ -50,38 +50,38 @@
       - [Overview](dev/how-to/deploy/geographic-redundancy/overview.md)
       - [Configure Location Awareness](dev/how-to/deploy/geographic-redundancy/location-awareness.md)
     - [TiSpark](dev/how-to/deploy/tispark.md)
-    - [Data Migration with Ansible](/tools/dm/deployment.md)
+    - [Data Migration with Ansible](dev/how-to/deploy/data-migration-with-ansible.md)
   + Configure
     - [Time Zone](sql/time-zone.md)
     - [Memory Control](sql/tidb-memory-control.md)
   + Secure
     + Transport Layer Security (TLS)
-      - [Enable TLS For MySQL Clients](sql/encrypted-connections.md)
-      - [Enable TLS Between TiDB Components](op-guide/security.md)
-    - [Generate Self-signed Certificates](op-guide/generate-self-signed-certificates.md)
+      - [Enable TLS For MySQL Clients](dev/how-to/secure/enable-tls-clients.md)
+      - [Enable TLS Between TiDB Components](dev/how-to/secure/enable-tls-between-components.md)
+    - [Generate Self-signed Certificates](dev/how-to/secure/generate-self-signed-certificates.md)
   + Monitor
-    - [Overview](op-guide/monitor-overview.md)
-    - [Monitor a TiDB Cluster](op-guide/monitor.md)
+    - [Overview](dev/how-to/monitor/overview.md)
+    - [Monitor a TiDB Cluster](dev/how-to/monitor/monitor-a-cluster.md)
   + Migrate
-    - [Overview](op-guide/migration-overview.md)
+    - [Overview](dev/how-to/migrate/overview.md)
     + Migrate from MySQL
-      - [Migrate the Full Data](op-guide/migration.md)
-      - [Migrate the Incremental Data](op-guide/migration-incremental.md)
+      - [Migrate the Full Data](dev/how-to/migrate/from-mysql.md)
+      - [Migrate the Incremental Data](dev/how-to/migrate/incrementally-from-mysql.md)
     - [Migrate from CSV](tools/lightning/csv.md)
   + Maintain
-    - [Backup and Restore](op-guide/backup-restore.md)
+    - [Backup and Restore](dev/how-to/maintain/backup-and-restore.md)
     + Scale
-      - [Scale a TiDB Cluster](op-guide/horizontal-scale.md)
-      - [Scale Using Ansible](op-guide/ansible-deployment-scale.md)
-    - [Identify Slow Queries](sql/slow-query.md)
+      - [Scale a TiDB Cluster](dev/how-to/maintain/scale/horizontally.md)
+      - [Scale using Ansible](dev/how-to/maintain/scale/with-ansible.md)
+    - [Identify slow queries](dev/how-to/maintain/identify-slow-queries.md)
```

Comment:
Shouldn't this be all initial-capped for top-level headers? (also applied to other similar changes)
```suggestion
    - [Identify Slow Queries](dev/how-to/maintain/identify-slow-queries.md)
```